packages/web-client/app/components/card-pay/layer-two-connect-card/index.ts

```diff
@@ -36,6 +36,8 @@ class CardPayLayerTwoConnectCardComponent extends Component<CardPayLayerTwoConne
   constructor(owner: unknown, args: CardPayLayerTwoConnectCardComponentArgs) {
     super(owner, args);
     if (this.isConnected) {
+      // this ends up running after rendering the whole workflow up to first incomplete postable
+      // which means test waiters start after they have been stopped
       next(this, () => {
         this.args.onComplete?.();
       });
```

Comment:
We use this pattern in other places too for cards that are completed from the start (eg. view only cards), would be good to review the use of `next` calling `onComplete` in the constructor - could lead to odd consequences.